src/install/install.zig

```diff
@@ -2144,9 +2144,9 @@ pub const PackageManager = struct {
 
         return std.fmt.allocPrint(
             this.allocator,
-            "https://{s}/repos/{s}/{s}{s}tarball/{s}",
+            "{s}/repos/{s}/{s}{s}tarball/{s}",
             .{
-                github_api_domain,
+                github_api_url,
```

Comment:
incase they add a trailing slash to the end of the URL:

```suggestion
                strings.withoutTrailingSlash(github_api_url),
```

Reply:
also technically needs to handle if they add a query param but hopefully they just don't do that